props/blaster.h

```diff
@@ -0,0 +1,192 @@
+#ifndef PROPS_BLASTER_H
+#define PROPS_BLASTER_H
+
+#include "prop_base.h"
+
+#define PROP_TYPE Blaster
+
+class Blaster : public PropBase {
+public:
+  Blaster() : PropBase() {}
+  const char* name() override { return "Blaster"; }
+
+#ifdef DELAYED_OFF

```

Comment:
What is DELAYED_OFF for?

Reply:
Delayed off was carried over from TD since I wanted to preserve self destruct

Reply:
didn't change this until I revisit the self-destruct.